views/courses/edit-course.hbs

```diff
@@ -280,7 +280,7 @@
 						</button>
 
                         {{> "lib/components/csrfInput"}}
-						<button type="submit" class="btn btn-primary btn-submit">
+						<button type="submit" class="btn btn-primary btn-submit" data-testid="submit_course_changes">
```

Comment:
as above with the same naming a specific id would be helpful to the context

Reply:
changed to modal_delete_course_button